### ISSUE TITLE
Vue3 codesandbox examples not working

### ISSUE THREAD

**Issue by alythal** [revolist/vue3-datagrid#2]
Many thanks for the amazing work on this project.

The examples you posted on codesandbox for Vue3 are not working.

1: https://codesandbox.io/s/revo-grid-vue3-thry7
2: https://codesandbox.io/s/revo-grid-vue3-forked-c16kb

<hr>

When I try to use the package locally it does not work either. I am using vitejs with the Vue 3 template.

Minimal project uploaded here: https://github.com/alythal/vue3-revolist-sample

The most important sections are pasted below:

App.vue
```vue
<template>
  <v-grid theme="compact" :source="rows" :columns="columns"></v-grid>
</template>


<script>
import { defineComponent } from "vue";
import VGrid from "@revolist/vue3-datagrid";
export default defineComponent({
  name: "App",
  data() {
    return {
      columns: [{ prop: "name" }, { prop: "details" }],
      rows: [
        {
          name: "1",
          details: "Item 1"
        }
      ]
    };
  },
  components: {
    VGrid
  }
});
</script>
```

package.json
```json
"dependencies": {
    "@revolist/vue3-datagrid": "^1.0.6",
    "vue": "^3.0.5"
  },
  "devDependencies": {
    "@vitejs/plugin-vue": "^1.1.2",
    "@vue/compiler-sfc": "^3.0.5",
    "@vuedx/typecheck": "^0.4.1",
    "@vuedx/typescript-plugin-vue": "^0.4.1",
    "typescript": "^4.1.3",
    "vite": "^2.0.0-beta.45"
  }
```

Firefox error message
![image](https://user-images.githubusercontent.com/44092779/108078083-86c90800-706d-11eb-8e12-1f48472cb2db.png)

Chromium error message
![image](https://i.imgur.com/mghq8fm.png)

If I could help with the issue please let me know, I tried debugging it myself but unfortunately with no success.

The issue [#5](https://github.com/revolist/vue-datagrid/issues/5) related to Vite seems to be fixed in the latest Vite.

**Comment by revolist** [comment]
@alythal, thank you for your feedback, it's very important for us.
I have tried examples in different browsers since you've noticed a problem.
Seems it's failing sometimes for Safari and Firefox. This issue totally internal for https://codesandbox.io.
It's recommended to try examples in chrome browser.  

https://github.com/codesandbox/codesandbox-client/issues/5366

I'll check your version in meantime.

**Comment by revolist** [comment]
Hello @alythal, starting from version 3.0 evertything should work just fine. I have updated your sample with working example:

https://github.com/revolist/vue3-revolist-sample
We have switched to esm modules and direct import.

But 3.0 still in beta, we are improving some things such as compatibility with modern frameworks, build systems such as vite and bootsrap support.

"@revolist/vue3-datagrid": "^3.0.0-beta.1"
